README.md

```diff
@@ -15,23 +15,35 @@
 ### Run app
 On IDEA create new gradle configuration and use `bootRun` as the run argument with env variables
```

Comment:
This section could described more explicitly for developers not so familiar with Gradle or Intellij IDEA

Reply:
Maybe add the link (https://www.jetbrains.com/help/idea/run-debug-gradle.html#add_advanced) to documentations etc